internal/fsx/fsx.go

```diff
@@ -0,0 +1,19 @@
+// Package fsx contains file system extension
+package fsx
+
+import (
+	"fmt"
+	"os"
+)
+
+func Open(name string) (*os.File, error) {
+	file, err := os.Open(name)
+	if err != nil {
+		return nil, err
+	}
+	info, err := file.Stat()
```

Comment:
```suggestion
	info, err := file.Stat()
	if err != nil {
		return nil, err
	}
```

We want to check the return value from stat before proceeding.